### PR TITLE
[Needs Review] b07902119 熊育霆

### PR DESCRIPTION
**Pull request by MrBear2000**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## What is the most challenging part in this homework?
Everything
## Is this homework helpful for you?
A little
## Is there any suggestion?
Too few instruction
<br />

### CODE REVIEW

**Comment by amjltc295** [comment]
This is a duplicate of #114. Do not open a new PR to fix problems.

File: src/students/b07902119.py [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    '''
+    from urllib import request
+    result_img = None
+
+    # TODO: download the image from img_url with the request module
+    # and add your student ID on it with draw_name() in the utils module
+    # under src/.
+    from PIL import Image
+    import sys
+    sys.path.append('~/PythonHomework/src')
```
Comment (amjltc295) [inline src/students/b07902119.py]: Don't hard-code the path. Besides, you probably don't need this line.

Comment (MrBear2000) [reply src/students/b07902119.py]: Done

**Comment by amjltc295** [comment]
-10 for sending three PRs.

**Pull request merged**